usecases/cluster/state.go

```diff
@@ -133,6 +133,9 @@ func (s *State) Hostnames() []string {
 
 // AllHostnames for live members, including self.
 func (s *State) AllHostnames() []string {
+	if s.list == nil {
```

Comment:
isn't handled now ? I mean it it's safe now it will return empty list which not gonna allow the recovery though

Reply:
it's the case now but in order to it has to wait https://github.com/weaviate/weaviate/pull/4087/files#diff-71a484f3dccfe2199cc13272f254d716ea12cdaf27ee8e29ef66ae5d5717f2abR171 so that is why it's checked on `recoverable()`